playwright/e2e/crypto/utils.ts

```diff
@@ -148,7 +148,7 @@ export async function logIntoElement(
     // select homeserver
     await page.getByRole("button", { name: "Edit" }).click();
     await page.getByRole("textbox", { name: "Other homeserver" }).fill(homeserver.config.baseUrl);
-    await page.getByRole("button", { name: "Continue" }).click();
+    await page.getByRole("button", { name: "Continue", exact: true }).click();
```

Comment:
For the new test added in this PR we are using a specific `default_server_config`, as a side effect there will be 2 buttons matching this query (Continue with oauth and just continue), so I added `exact` here